scripts/helm/publish-chart-yaml.sh

```diff
@@ -10,15 +10,89 @@
 # are changed to the appTag.
 # If the appTag is a stable release then both version and appVersions are changed to the appTag.
 
+# Write output to error output stream.
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
 die()
 {
   local _return="${2:-1}"
-  echo "$1" >&2
+  echo_stderr "$1"
   exit "${_return}"
 }
 
 set -euo pipefail
 
+# Requires a 'git fetch origin' (origin being the remote entry for openebs/mayastor-extensions) or equivalent.
+latest_release_branch() {
```

Comment:
Can't we just check the already fetched origin instead to avoid making this script requiring internet?

Reply:
This doesn't require the internet. This checks the existing origin refs. This comment just points out that a user has to do this, if they have done a very minimal clone.

Reply:
I've changed the comment to be a bit for clear on what this function does.